model/predict_on_image_sliding_window.py

```diff
@@ -0,0 +1,67 @@
+import time
+import cv2
+import numpy as np
+from ultralytics import YOLO
+from ultralytics.utils.plotting import Annotator
+from predict_on_image import MODEL_PATH, select_random_image
```

Comment:
Idk if good idea taip dependent ant kito failo. Bent ta MODEL_PATH reiktu kad cia atskiras butu. Arba gal atskira vieta config.py is kurio visi scriptai inference imtu configa bendra, ir gal tada helper funkcijom faila, bbz siaip ant kiek cia plesis reikalavimai.

Reply:
Fixed. Iškėliau select_random_image į helpers.py.
Šiaip irgi idk, kiek čia verta taip db žaist, kai nejsn, ko norės ant ant galo